packages/swing-store/src/bundleStore.js

```diff
@@ -172,8 +229,10 @@ export function makeBundleStore(db, ensureTxn, noteExport = () => {}) {
     // prettier-ignore
     (parts.length === 2 && type === 'bundle') ||
       Fail`expected artifact name of the form 'bundle.{bundleID}', saw ${q(name)}`;
-    const rawBundle = sqlGetBundle.get(bundleID);
-    rawBundle || Fail`bundle ${q(name)} not available`;
+    const row = sqlGetBundle.get(bundleID);
+    row || Fail`bundle ${q(bundleID)} not found`;
+    const rawBundle = row.bundle;
+    rawBundle || Fail`bundle ${q(bundleID)} pruned`;
```

Comment:
Potentially:
```suggestion
    const row = sqlGetBundle.get(bundleID) || Fail`bundle ${q(bundleID)} not found`;
    const rawBundle = row.bundle || Fail`bundle ${q(bundleID)} pruned`;
```
or even abstract this into a helper for use by both `getBundle` and `exportBundle`.

Reply:
I'll apply the change, but I don't think a two-line function is worth factoring out.